pulp_rpm/app/serializers/repository.py

```diff
@@ -147,11 +146,26 @@ class RpmRepositorySerializer(RepositorySerializer):
         ),
         read_only=True,
     )
-    repo_config = CustomJSONField(
+    repo_config = serializers.JSONField(
         required=False,
         help_text=_("A JSON document describing config.repo file"),
     )
 
+    def to_representation(self, instance):
+        data = super().to_representation(instance)
+        # Import workflow may cause these fields to be stored as "" in the database
```

Comment:
Is there a separate codepath in the import workflow that ought to be adjusted here, or does it all feed back through this serializer?

Reply:
My understanding is that we can fix import-export workflow to ensure it stores null values and add migration to fix empty strings. I can open an issue for that.